components/labeler/RecordView.tsx

```diff
@@ -0,0 +1,352 @@
+]
+
+const subjectTypes = ['account', 'record', 'chat']
+
+const LabelDefinitionView = ({
+  label,
+  definition,
+  onUpdate,
+}: {
+  label: ComAtprotoLabelDefs.LabelValue
+  definition?: ComAtprotoLabelDefs.LabelValueDefinition
+  onUpdate: (
+    label,
+    updatedDef?: ComAtprotoLabelDefs.LabelValueDefinition,
+  ) => void
+}) => {
+  const [isEditing, setIsEditing] = useState(false)
+
+  if (isEditing) {
+    return (
+      <LabelDefinitionEditor
+        onCancel={() => setIsEditing(false)}
+        definition={definition}
+        label={label}
+        onUpdate={(label, updatedDef) => {
+          onUpdate(label, updatedDef)
+          setIsEditing(false)
+        }}
+      />
+    )
+  }
+
+  if (!definition) {
+    return (
+      <div className="mb-1 border-b border-gray-300 dark:border-gray-700 pb-2">
+        <LabelChip className="ml-0">{label}</LabelChip>
+        <button
+          onClick={() => setIsEditing(true)}
+          type="button"
+          title={`Edit ${label} label definition`}
+        >
+          <PencilSquareIcon className="w-3 h-3 ml-1" />
+        </button>
+      </div>
+    )
+  }
+
+  return (
+    <div className="mb-1 border-b dark:border-gray-700 border-gray-300 pb-2">
+      <div>
+        <LabelChip className="ml-0">{definition.identifier}</LabelChip>
+        {definition.adultOnly && <span className="text-sm">🔞 Adult Only</span>}
+        <button
+          onClick={() => setIsEditing(true)}
+          type="button"
+          title={`Edit ${definition.identifier} label definition`}
+        >
+          <PencilSquareIcon className="w-3 h-3 ml-1" />
+        </button>
+      </div>
+      <div className="flex flex-row items-center mt-2">
+        <p className="text-sm text-gray-500 dark:text-gray-400 flex flex-row items-center border-r dark:border-gray-600 pr-2 mr-2">
+          <ExclamationCircleIcon className="h-4 w-4 mr-1" />
+          Severity:
+          <span className="capitalize text-gray-700 dark:text-gray-300 ml-1">
+            {definition.severity}
+          </span>
+        </p>
+        <p className="text-sm text-gray-500 dark:text-gray-400 flex flex-row items-center border-r dark:border-gray-600 pr-2 mr-2">
+          <EyeSlashIcon className="h-4 w-4 mr-1" />
+          Blurs:
+          <span className="capitalize text-gray-700 dark:text-gray-300 ml-1">
+            {definition.blurs}
+          </span>
+        </p>
+        <p className="text-sm text-gray-500 dark:text-gray-400">
+          Default Setting:
+          <span className="capitalize text-gray-700 dark:text-gray-300 ml-1">
+            {definition.defaultSetting}
+          </span>
+        </p>
+      </div>
+
+      <div className="mt-2">
+        <h3 className="text-sm font-medium text-gray-800 dark:text-gray-200">
+          Locales
+        </h3>
+        <div className="mt-2 space-y-2">
+          {definition.locales.map((locale, index) => (
+            <div
+              key={index}
```

Comment:
Everything is stateless in here so probably no biggie, but using `index` as a key can be a footgun.  Would `locale.lang` do?